tools/tertiary-analysis/scanpy/scanpy-run-pca.xml

```diff
@@ -40,19 +40,19 @@ PYTHONIOENCODING=utf-8 scanpy-run-pca.py
   <inputs>
     <expand macro="input_object_params"/>
     <expand macro="output_object_params"/>
-    <param name="n_pc" argument="--n-pcs" type="integer" value="50" label="Number of PCs to produce"/>
+    <param name="n_pcs" argument="--n-pcs" type="integer" value="50" label="Number of PCs to produce"/>
     <conditional name="run_mode">
       <param name="chunked" argument="--chunked" type="boolean" checked="false" label="Perform incremental PCA by chunks"/>
       <when value="true">
         <param name="chunk_size" argument="--chunk-size" type="integer" value="0" label="Chunk size"/>
       </when>
       <when value="false">
-        <param name="zero_center" argument="--zero-center" type="boolean" checked="true" label="Zero center data before scaling"/>
```

Comment:
I would revert this one, as scanpy-scripts uses --zero-center, we can fix the parameter on nextflow for this one.